src/components/Form.js

```diff
@@ -1,12 +1,41 @@
+import { useState } from 'react';
+import { useDispatch } from 'react-redux';
+import { addBook } from '../redux/books/bookSlice';
+
 function Form() {
+  const dispatch = useDispatch();
+  const [book, setbook] = useState({
+    title: '', item_id: '', author: '', category: 'fiction',
+  });
+  const handlesubmit = (e) => {
+    e.preventDefault();
+    dispatch(addBook({ next: book }));
+    setbook({ title: '', item_id: '', author: '' });
+  };
+
   return (
     <div>
-      <form className="newBook">
+      <form className="addNew" onSubmit={handlesubmit}>
```

Comment:
- [Optional] This event handler was supposed to be implemented on the button below(onClick) and not on the form itself. You can check the requirement once more.